src/wiki/templates/wiki/permission_denied.html

```diff
@@ -17,10 +17,20 @@ <h1 class="page-header">{% trans "Permission Denied" %}</h1>
   </p>
   {% endif %}
 
+  {% if not read_denied %}
   <p>
     <a href="{% url 'wiki:get' article_id=article.id path=urlpath.path %}" class="btn btn-default">
       {% trans "Back to article" %}
     </a>
   </p>
+  {% else %}
```

Comment:
Instead of nesting else and if I think an elif would be better. 

```django
  {% elif urlpath.parent %}
    <p>
      <a href="{% url 'wiki:get' path=urlpath.parent.path %}" class="btn btn-default">
        {% trans "Back to article" %}
      </a>
    </p>
  {% endif %}
```